### DISCUSSION TITLE
Error while running gmx_MMPBSA

### DISCUSSION THREAD

**Discussion by drbipinraj** [Valdes-Tresanco-MS/gmx_MMPBSA#378] [General]
I am trying to run gmxMMPBSA for GB calculation but the program gives the following error kindly guide me through it.
the command which I have given is:  mpirun -np 2 gmx_MMPBSA -O -i mmpbsa.in  -cs MD.tpr -ci index.ndx -cg 1 13 -ct MD.xtc -cp topol.top -o FINAL_RESULTS_MMPBSA.dat -eo FINAL_RESULTS_MMPBSA.csv

I have used   gmx_MMPBSA --create_input gb   for creating .in file

The .in file and the log file is attached also the snippet of the run information and error message is attached
[gmx_MMPBSA.log](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/11608675/gmx_MMPBSA.log)
[gmx_MMPBSA.log](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/11608679/gmx_MMPBSA.log)
[mmpbsa.txt](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/11608693/mmpbsa.txt)
[error gmxMMPBSA.txt](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/11608695/error.gmxMMPBSA.txt)

**Comment by Valdes-Tresanco-MS** [comment]
Probably your topology is inconsistent. Please, check there is not space before the #. If you can't solve the problem, attach the topology and the itps

**Reply by drbipinraj** [reply]
The error was in the topology file.
There was space before # in the topology file, After removing the space the error was resolved.
Thank you very much.
